roxie/udplib/udptopo.cpp

```diff
@@ -323,6 +334,32 @@ StringBuffer &CTopologyServer::report(StringBuffer &ret) const
     return ret;
 }
 
+void CTopologyServer::updateStatus() const
+{
+    // Set the k8s ready probe status according to whether we have at least one agent available per channel
+    bool ready = true;
+    for (unsigned channel=1; channel <= numChannels; channel++)
+    {
+        if (!queryAgents(channel).length())
+        {
+            DBGLOG("channel %u has no agents available", channel);
+            ready = false;
+            //break;
```

Comment:
minor: Did you want to reinstate this break, otherwise you might get lots of tracing.

Reply:
I suspect I should either remove the tracing and re-add the break, or leave it as is.

Reply:
Reworked it a bit